R/validation fonctions AE_table_grade/group_grades_zeroNA.R

```diff
@@ -0,0 +1,23 @@
+#fonction grouping figures of missing grades and grades 0 for R function ae_table_grade
+group_grades_zeroNA <- function(data, round = 0){
+  
+  #suming N of missing grades and grades 0
+  data <- data %>%
+    mutate(grade = replace_na(grade, 0)) %>% 
+    group_by(grade) %>% 
+    mutate(across(starts_with("N"), ~sum(., na.rm = T))) %>% 
+    distinct(grade, .keep_all = T) %>% 
+    ungroup()
+  
+  #recalculating pct
+  ngroups <- (ncol(data) - 1) / 2
+  for(i in 1:ngroups){
```

Comment:
I wanted to use purrr but I struggled to find an optimal way to use it in this case